templates/devices.html

```diff
@@ -27,4 +27,9 @@ <h2>Currently supported</h2>
 			{% endif %}
 		{% endfor %}
 
+<<<<<<< HEAD
```

Comment:
@joehounsham I assume these were left over when trying to sort out merge conflicts?

Reply:
The change here was modifying the contact email to solutions@balena.io

Reply:
@joehounsham I was more refering to the `<<<<<<< HEAD` which I assume you didn't plan on including?